.eslintrc.json

```diff
@@ -0,0 +1,192 @@
+{
+	"root": true,
+	"parser": "@typescript-eslint/parser",
+	"plugins": ["@typescript-eslint", "prettier", "simple-import-sort"],
+	"extends": ["eslint:recommended", "plugin:@typescript-eslint/eslint-recommended", "plugin:@typescript-eslint/recommended", "prettier"],
+	"rules": {
+		"no-useless-catch": "error", //TODO- Remove the unnecessary try-catch lines used and enable no-useless-catch rule.
```

Comment:
nit: technically .json doesn't accept comments. jsonc does. But I'm not sure whether eslint accepts it.

Reply:
Removed the comments. I added few comments to separate the type of rules. Removed those as well.